app/components/RegisterCard/index.js

```diff
@@ -1,34 +1,92 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { client } from '../../graphql/apolloClient';
+
+async function doMutation(registerBody) {
+	const { data } = await client.mutate({
+		mutation: registerUser,
+		variables: {
+			registerBody: registerBody,
+		},
+	});
+	return data
+}
 
 export const RegisterCard = () => {
+	const [username, setUsername] = useState();
```

Comment:
```suggestion
	const [username, setUsername] = useState('');
```
Definir estados iniciales